pkg/workceptor/kubernetes.go

```diff
@@ -703,6 +718,14 @@ func (kw *kubeUnit) SetFromParams(params map[string]string) error {
 	if userPod != "" && (userParams != "" || userCommand != "" || userImage != "") {
 		return fmt.Errorf("params kube_command, kube_image, kube_params not compatible with secret_kube_pod")
 	}
+
+	if pod_pending_timeout_string, ok := params["pod_pending_timeout"]; ok {
```

Comment:
we still need to go through the setter above, like all the other params
```golang
pod_pending_timeout_string := ""
.
.
{name: "pod_pending_timeout", permission: kw.allowRuntimeParams, setter: setString(&pod_pending_timeout_string)},

```
then you can set `kw.podPendingTimeout` to `pod_pending_timeout_string`

Reply:
I have made the required changes.